www/src/data/sidebars/doc-links.yaml

```diff
@@ -333,7 +333,7 @@
           items:
             - title: Creating Prefixed 404 Pages for Different Languages
               link: /docs/creating-prefixed-404-pages-for-different-languages/
-    - title: API Reference
+    - title: Gatsby API
```

Comment:
Should we keep "reference" in here?

Reply:
I went back and forth on that, and ultimately decided to differentiate from "Reference Guides". But we could change it!

Reply:
I'll defer to you here. It's easy to change later if we see people getting lost.